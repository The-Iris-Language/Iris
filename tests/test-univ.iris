.. test univ keyword included
class Main () {
  public: 
  int univ main() {
    int i;
    int j;

    i = 1;
    return 0;
  }
}


