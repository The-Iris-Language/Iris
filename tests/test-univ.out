class Main() {
int univ main() {
int i;
int j;
i = 1;
return 0;
}
}